task/buildah/0.6/buildah.yaml

```diff
@@ -568,6 +575,14 @@ spec:
         BUILDAH_ARGS+=("--inherit-labels=false")
       fi
 
+      if [ -n "${BUILDAH_SOURCE_DATE_EPOCH}" ]; then
+        BUILDAH_ARGS+=("--source-date-epoch=${BUILDAH_SOURCE_DATE_EPOCH}")
+        BUILDAH_ARGS+=("--rewrite-timestamp")
+        if [ -z "$BUILD_TIMESTAMP" ]; then
+          BUILD_TIMESTAMP="$BUILDAH_SOURCE_DATE_EPOCH"
```

Comment:
Since we already have the `BUILD_TIMESTAMP` param, WDYT about using its value instead of adding a new param? Adding a new param would:
- create an opportunity for mismatched timestamps
- make it so that the existing users of BUILD_TIMESTAMP don't benefit from your changes

Reply:
Sigh, sorry, your second comment appeared after I commented and some React websocket on the page decided to reactivate and render it. :)

> That's enough of a difference to warrant a separate parameter. Can we instead make BUILD_TIMESTAMP and SOURCE_DATE_EPOCH mutually exclusive?

Yeah, that's probably fair I guess. Let me look into this.

Reply:
>> That's enough of a difference to warrant a separate parameter. Can we instead make BUILD_TIMESTAMP and SOURCE_DATE_EPOCH mutually exclusive?
>
> Yeah, that's probably fair I guess. Let me look into this.

OK did that! Still need to test it. I did though still made it set the `BUILD_TIMESTAMP` variable so that we get all the non-standard labels like `build-date` to agree.

Reply:
OK yup, that check works as expected! :heavy_check_mark: